build/_build/Build.Steps.cs

```diff
@@ -363,30 +363,33 @@ partial class Build
     /// <summary>
     /// This target is a bit of a hack, but means that we actually use the All CPU builds in intgration tests etc
     /// </summary>
-    Target CopyPlatformlessBuildOutput => _ => _
+    Target CreatePlatformlessSymlinks => _ => _
         .Description("Copies the build output from 'All CPU' platforms to platform-specific folders")
         .Unlisted()
         .After(CompileManagedSrc)
         .After(CompileDependencyLibs)
-        .After(CompileManagedUnitTests)
+        .After(CompileManagedTestHelpers)
         .Executes(() =>
         {
+            // create junction for each directory
             var directories = RootDirectory.GlobDirectories(
                 $"src/**/bin/{BuildConfiguration}",
                 $"tools/**/bin/{BuildConfiguration}",
                 $"test/Datadog.Trace.TestHelpers/**/bin/{BuildConfiguration}",
                 $"test/test-applications/integrations/dependency-libs/**/bin/{BuildConfiguration}"
             );
-            directories.ForEach(source =>
+
+            directories.ForEach(existingDir =>
             {
-                var target = source.Parent / $"{Platform}" / BuildConfiguration;
-                if (DirectoryExists(target))
+                var newDir = existingDir.Parent / $"{Platform}" / BuildConfiguration;
+                if (DirectoryExists(newDir))
                 {
-                    Logger.Info($"Skipping '{target}' as already exists");
+                    Logger.Info($"Skipping '{newDir}' as already exists");
                 }
                 else
                 {
-                    CopyDirectoryRecursively(source, target, DirectoryExistsPolicy.Fail, FileExistsPolicy.Fail);
+                    EnsureExistingDirectory(newDir.Parent);
+                    Cmd.Value(arguments: $"cmd /c mklink /J \"{newDir}\" \"{existingDir}\"");
```

Comment:
Does the cmd command work on Linux?

Reply:
It doesn't need to, this one is only called in `BuildWindowsIntegrationTests` 🙂

Yes, it kinda sucks that we have completely different builds on each architecture 😉